ffi/harfbuzz_coverage.lua

```diff
@@ -0,0 +1,359 @@
+-- Generated by https://github.com/ezdiy/unimap/blob/master/gen.lua
+-- Each row is a sequence of {delta,count} pairs, where
+--   * `delta` denotes start of next range since the end of the previous one
+--   * `count` is number of glyphs in range
+-- For instance set of ranges:
+-- 0041..005A, 0061..007A, 00AA, 00BA ...
+-- Becomes:
+-- 65,26, 8,26, 49,1 17,1 ...
+return {
+    -- Table of ranges for entire scripts, ordered by UCDN_SCRIPT_* macros, starting with UCDN_SCRIPT_COMMON
+    -- Source data is https://www.unicode.org/Public/5.2.0/ucd/Scripts.txt
+    -- UCDN is internally used to map character classes (and detect scripts) both by harfbuzz and mupdf
```

Comment:
Thanks for the comments.
No risk these macros/indices could get out of sync between HB and MuPDF if a new script is introduced between others?

Reply:
In mu/hb, the order is hardcoded *without* regard for the source data (that is compiled into UC DB contents later). So I'm essentially doing the same here (in gen.lua). The symbolic indices aren't all that interesting to us - mupdf tells us this indice directly as int, for HB its a bit awkward because they translate the UCD canonical order to their own `hb_script_t`, maintaining a lookup table between the two which sucks. Luckily, these indices are probably never gonna be used with HB directly.